code/game/gamemodes/cult/runes.dm

```diff
@@ -92,8 +94,10 @@ structure_check() searches for nearby cultist structures required for the invoca
 /obj/effect/rune/proc/can_invoke(var/mob/living/user)
 	//This proc determines if the rune can be invoked at the time. If there are multiple required cultists, it will find all nearby cultists.
 	if(!structure_check(req_pylons, req_forges, req_archives, req_altars))
+		user << "<span class='warning'>[src] lacks the nearby structures it requires!</span>"
 		return 0
 	if(!keyword && req_keyword)
+		user << "<span class='warning'>[src] requires no keyword!</span>"

```

Comment:
requires a keyword, not no keyword.
though given that this won't normally appear, eh.